packaging/language/bower.py

```diff
@@ -48,6 +48,11 @@
     description:
       - The base path where to install the bower packages
     required: true
+  relative_execpath:
+    description:
+      - Relative path to bower executable from install path
```

Comment:
Maybe a bit clearer if it was something like: `Relative path to bower executable from install path (if bower is not installed globally)`

Might help people understand when they would need to use this?